README.md

```diff
@@ -467,131 +464,137 @@ $ kogito new-project kogito-cli
-1. `namespace` (required) is a given namespace where the test will run.
-2. `tag` (optional, default is current release) is the image tag for the Kogito image builds, for example: `0.6.0-rc1`. Useful on situations where [Kogito Cloud images](https://github.com/kiegroup/kogito-cloud/tree/master/s2i) haven't released yet and are under a temporary tag
-3. `native` (optional, default is `false`) indicates if the e2e test should use native or jvm builds. See [Native X JVM Builds](#native-x-jvm-builds)
-4. `maven_mirror` (optional, default is empty) the Maven mirror URL. Useful when you need to speed up the build time by referring to a closer maven repository
-5. `skip_build` (optional, default is `true`) set to `true` to skip building the CLI before running the test
+- `namespace` (required) is a given temporary namespace where the test will run.
+- `tag` (optional, default is current release) is the image tag for the Kogito image builds, for example, `0.6.0-rc1`. This is helpful in situations where [Kogito S2I images](https://github.com/kiegroup/kogito-cloud/tree/master/s2i) have not been released yet and are under a temporary tag.
+- `native` (optional, default is `false`) indicates whether the E2E test should use `native` or `jvm` builds. For more information, see [Native X JVM builds](#native-x-jvm-builds).
+- `maven_mirror` (optional, default is empty) is the Maven mirror URL. This is helpful when you need to speed up the build time by referring to a closer Maven repository.
+- `skip_build` (optional, default is `true`) is set to `true` to skip building the CLI before running the test.
 
-### Running Locally
+### Running the Kogito Operator locally
 
-Change log level at runtime with the `DEBUG` environment variable. e.g. -
+To run the Kogito Operator locally, change the log level at runtime with the `DEBUG` environment variable, as shown in the following example:
 
 ```bash
 $ make mod
 $ make clean
 $ DEBUG=true operator-sdk up local --namespace=<namespace>
 ```
 
-Before submitting PR, please be sure to read the [contributors guide](docs/CONTRIBUTING.MD).
+Before submitting a [pull request](https://help.github.com/en/articles/about-pull-requests) to the Kogito Operator repository, review the instructions for [Contributing to the Kogito Operator](docs/CONTRIBUTING.MD).
 
-It's always worth noting that you should generate, vet, format, lint, and test your code. This all can be done with one command.
+You can use the following command to vet, format, lint, and test your code:
 
 ```bash
 $ make test
 ```
 
-## Prometheus Integration
+## Prometheus integration with the Kogito Operator
+
+### Prometheus annotations
 
-### Prometheus Annotations
+By default, if your Kogito Runtimes service contains the `monitoring-prometheus-addon` dependency, metrics for the Kogito service are enabled. For more information Prometheus metrics in Kogito services, see [Enabling metrics](https://github.com/kiegroup/kogito-runtimes/wiki/Configuration#enabling-metrics).
```

Comment:
For more information _about_ Prometheus metrics in Kogito services?

Reply:
Oops! Fixed. Thanks.